src/content_mapblock.cpp

```diff
@@ -330,65 +245,37 @@ static void makeSmoothLightedCuboid(MeshCollector *collector, const aabb3f &box,
 			break;
```

Comment:
^ For lines 200 to 241 above (in original code): spaces around * + and after commas.